metashare/api/models.py

```diff
@@ -258,10 +273,40 @@ def get_serialized_representation(self):
 
     # end PushMixin configuration
 
+    # begin CreatePrMixin configuration:
+    create_pr_event = "PROJECT_PR_CREATE"
+
+    def get_repo_id(self, user):
+        return self.repository.get_repo_id(user)
+
+    def get_base(self):
+        return self.branch_name
+
+    def get_head(self):
+        return self.repository.branch_name
```

Comment:
These are backwards, I think. `base` should be the repo branch, `head` should be the project branch.